eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -490,16 +496,15 @@ object Helpers {
       * @param commitments our commitment data, which include payment preimages
       * @return a list of transactions (one per HTLC that we can claim)
       */
-    def claimCurrentLocalCommitTxOutputs(keyManager: KeyManager, commitments: Commitments, tx: Transaction)(implicit log: LoggingAdapter): LocalCommitPublished = {
+    def claimCurrentLocalCommitTxOutputs(feeTargets: FeeTargets, keyManager: KeyManager, commitments: Commitments, tx: Transaction, feeEstimator: FeeEstimator)(implicit log: LoggingAdapter): LocalCommitPublished = {
```

Comment:
`feeTargets` should be after `feeEstimator`

Reply:
Done in f0649dd